tools/pipelines/deploy-docs.yml

```diff
@@ -0,0 +1,266 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
```

Comment:
Nit: since this pipeline is specifically about deploying the website, what do you think about `deploy-website.yml`?